__Main__.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<p>Nice work using the validation set! Lot's of people forgot this lol. Remember why it is necessary?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI-Core-Assignments/project-1-Farhad1353/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>Certainly,</p><p>I like the validation set, as it helps me to see how the model is(maybe overfit or underfit) at the same time that I'm training the model and helps to stop or change the direction of training if needed.</p>